src/parser/tptp/Tptp.g

```diff
@@ -1422,79 +1419,89 @@ tffUnitaryFormula[CVC4::Expr& expr]
     RPAREN_TOK
   ;
 
-tffLetTermDefn[CVC4::Expr& lhs, CVC4::Expr& rhs]
+tffLetTermDefn[CVC4::api::Term& lhs, CVC4::api::Term& rhs]
 @declarations {
-  std::vector<CVC4::Expr> bvlist;
+  std::vector<CVC4::api::Term> bvlist;
 }
   : (FORALL_TOK LBRACK_TOK tffVariableList[bvlist] RBRACK_TOK COLON_TOK)*
     tffLetTermBinding[bvlist, lhs, rhs]
   ;
 
-tffLetTermBinding[std::vector<CVC4::Expr>& bvlist, CVC4::Expr& lhs, CVC4::Expr& rhs]
+tffLetTermBinding[std::vector<CVC4::api::Term>& bvlist, CVC4::api::Term& lhs, CVC4::api::Term& rhs]
   : term[lhs] EQUAL_TOK term[rhs]
     { PARSER_STATE->checkLetBinding(bvlist, lhs, rhs, false);
-      rhs = MK_EXPR(kind::LAMBDA, MK_EXPR(kind::BOUND_VAR_LIST, lhs.getChildren()), rhs);
-      lhs = lhs.getOperator();
+      std::vector<api::Term> lchildren;
+      for( unsigned i=1, nchild = lhs.getNumChildren(); i<nchild; i++)
+      {
+        lchildren.push_back(lhs[i]);
+      }
```

Comment:
Maybe like this?

```suggestion
      std::vector<api::Term> lchildren(lhs.begin() + 1, lhs.end());
```

Reply:
I had tried that, I get this compilation error:
```
/tmp/ajreynol/cmake/ajr/prod/src/parser/tptp/TptpParser.c: In function ‘void tffLetTermBinding(pTptpParser, std::vector<CVC4::api::Term>&, CVC4::api::Term&, CVC4::api::Term&)’:
/tmp/ajreynol/cmake/ajr/prod/src/parser/tptp/TptpParser.c:11720:73: error: no match for ‘operator+’ (operand types are ‘CVC4::api::Term::const_iterator’ and ‘int’)
11720 |                            std::vector<api::Term> lchildren(lhs.begin() + 1, lhs.end());

```

Reply:
Does `++lhs.begin()` work?

Reply:
Yes, I'll update.